app/services/catalog/import_service_plans.rb

```diff
@@ -2,10 +2,9 @@ module Catalog
   class ImportServicePlans
     attr_reader :json
 
-    def initialize(portfolio_item_id)
+    def initialize(portfolio_item_id, force_reset=false)
```

Comment:
Only thing I might suggest is to make `force_reset` a keyword arguments.  Then it would be really clear from the caller what `true` was actually doing.  But I'm ok with whatever you decide to use.